docs/content/guides/developer/sui-101/create-coin.mdx

```diff
@@ -4,26 +4,177 @@ title: Create a Coin
 
 Publishing a coin on Sui is nearly as straightforward as publishing a new type. The main difference is the requirement of a [one-time witness](/concepts/sui-move-concepts/one-time-witness.mdx) when creating a coin.
 
-```move
+```rust
```

Comment:
Why the change here? I thought we had move highlighting? Or do we not? :(

Reply:
We used to not. Looks like the SF team added, so thanks for pointing out.